src/handlers.rs

```diff
@@ -701,3 +702,21 @@ fn handle_symlink(i: *const isolate, base: &msg::Base) -> Box<Op> {
     }()))
   }
 }
+
+fn handle_truncate(i: *const isolate, base: &msg::Base) -> Box<Op> {
+  let deno = from_c(i);
+  if !deno.flags.allow_write {
+    return odd_future(permission_denied());
+  }
+   let msg = base.msg_as_truncate().unwrap();
```

Comment:
Remove this extra space (you might want to run `./tools/format.py` and `./tools/lint.py`)